script.js

```diff
@@ -51,3 +52,275 @@ forBtn.addEventListener('click', () => {
   bodyCo.style.overflow = 'auto';
   nav.style.width = 'auto';
 });
+
+/* Works Session */
+
+const worksCardArr = [
+  {
+    Id: 1,
+    Svg: ['./img/cardPlaholderM.png', './img/cardPlaholderD.png'],
+    Img: ['./img/m-Snapshoot-Portfolio.svg', './img/Snapshoot-Portfolio.svg'],
```

Comment:
- [ ] I like how you are using different images for desktop and mobile versions. Great job. :+1: